src/components/ui/Button.tsx

```diff
@@ -19,19 +19,28 @@ export const ButtonWithState: React.FC<LoadableButtonProps> = ({
   onClick,
   children,
 }) => {
+  const [internalLoading, setInternalLoading] = React.useState(false);
```

Comment:
Can we remove `React.` for code style

Reply:
Done, and I finally understood where this comes from: GitHub Copilot writes this style by default! We can change other occurrences in later PRs.